.github/workflows/main.yml

```diff
@@ -68,12 +69,19 @@ jobs:
           name: ${{ env.MACHINE_EMULATOR_TOOLS_TAR_GZ }}
           path: ${{ env.MACHINE_EMULATOR_TOOLS_TAR_GZ }}
 
+      - name: Upload [${{ env.MACHINE_EMULATOR_TOOLS_DEB }}]
```

Comment:
I downloaded and extracted the artifact 'machine-emulator-tools-v0.11.0.deb' from CI (link https://github.com/cartesi/machine-emulator-tools/suites/13594326107/artifacts/749193313 ) . And it is empty.

Reply:
You are correct!

Made some mistake when reworking the previous PR.
Let me fix that.